app/src/components/misc/ProductWalkthrough/index.tsx

```diff
@@ -11,18 +16,24 @@ import {
 interface TourProps {
   startWalkthrough: boolean;
   tourFor: string;
+  context: any;
   runTourWithABTest: boolean; //temporary flag
 }
 
 export const ProductWalkthrough: React.FC<TourProps> = ({
   startWalkthrough = false,
   tourFor,
+  context,
   runTourWithABTest = true,
 }) => {
   const [hasReachedLastStep, setHasReachedLastStep] = useState<boolean>(false);
   const joyrideRef = useRef(null);
   const WalkthroughHelpers = joyrideRef.current?.WalkthroughHelpers;
 
+  const renderCustomToolTip = (props: TooltipRenderProps) => {
```

Comment:
please memoize using `useCallback`, as it is being passed as prop to other component.